package/upgrade/Dockerfile

```diff
@@ -15,8 +15,8 @@ RUN curl -sfL https://storage.googleapis.com/kubernetes-release/release/${KUBECT
     chmod +x /usr/bin/kubectl
 
 RUN curl -sfL https://github.com/kubevirt/kubevirt/releases/download/v0.55.2/virtctl-v0.55.2-linux-${ARCH} -o /usr/bin/virtctl && chmod +x /usr/bin/virtctl && \
-    curl -sfL https://github.com/mikefarah/yq/releases/latest/download/yq_linux_${ARCH} -o /usr/bin/yq && chmod +x /usr/bin/yq && \
-    curl -sfL https://github.com/rancher/wharfie/releases/latest/download/wharfie-amd64  -o /usr/bin/wharfie && chmod +x /usr/bin/wharfie
+    curl -sfL https://github.com/mikefarah/yq/releases/download/v4.35.2/yq_linux_${ARCH} -o /usr/bin/yq && chmod +x /usr/bin/yq && \
```

Comment:
let's keep the yq one (it fixes CVE issues ...)

Reply:
Reverted it. Thank you.